validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -501,6 +530,37 @@ private AttesterDuties getAttesterDutiesFromIndexesAndState(
         });
   }
 
+  private SyncCommitteeDuties getSyncCommitteeDutiesFromIndexesAndState(
+      final BeaconState state,
+      final SyncCommitteeUtil syncCommitteeUtil,
+      final UInt64 epoch,
+      final List<Integer> validatorIndices) {
+    return new SyncCommitteeDuties(
+        validatorIndices.stream()
+            .map(
+                validatorIndex ->
+                    getSyncCommitteeDuty(state, syncCommitteeUtil, epoch, validatorIndex))
+            .flatMap(Optional::stream)
```

Comment:
nit: I'd collapse these:
```suggestion
            .flatMap(
                validatorIndex ->
                    getSyncCommitteeDuty(state, syncCommitteeUtil, epoch, validatorIndex).stream())
```